fabrics.c

```diff
@@ -643,7 +643,7 @@ char *nvmf_hostid_from_hostnqn(const char *hostnqn)
 
 void nvmf_check_hostid_and_hostnqn(const char *hostid, const char *hostnqn, unsigned int verbose)
 {
-	char *hostid_from_file, *hostid_from_hostnqn;
+	char *hostid_from_file = NULL, *hostid_from_hostnqn = NULL;
```

Comment:
Let's  use the `_cleanup_free_` helpers here as well.

Reply:
Sure, will do.